infra/app-rails/service/main.tf

```
@@ -125,9 +125,12 @@ module "service" {
     } : {},
   )
 
+  ephemeral_write_volumes = [
+    "/rails/tmp"
+  ]
```

can we move this to service config (currently in app-config/env-config/outputs.tf even though we should probably move it to it's own file app-config/env-config/service.tf like we do with other config objects) so that we can minimize the number of customizations to the root modules

```suggestion
  ephemeral_write_volumes = local.service_config.ephemeral_write_volumes
```